src/main/resources/Macros/MapMacro/Code/LeafletMap.xml

```diff
@@ -175,10 +176,14 @@
 
       var marker = L.marker(latLng, {icon: icon}).addTo(mymap);
       marker.on('click', onClick);
+      if(info != '')
```

Comment:
Not a big deal but in general at XWiki we prefer using brackets for `if` statements even when the block contains a single line, at least when the block is in a new line (I know this JavaScript file contains some parts that do not comply with this rule for now yet though).

Reply:
That is:
```
if (info != '') {
 ....
}
```